actors/builtin/miner/monies.go

```diff
@@ -61,17 +61,26 @@ func PledgePenaltyForUndeclaredFault(rewardEstimate, networkQAPowerEstimate *smo
 
 // Penalty to locked pledge collateral for the termination of a sector before scheduled expiry.
 // SectorAge is the time between the sector's activation and termination.
-func PledgePenaltyForTermination(dayRewardAtActivation, twentyDayRewardAtActivation abi.TokenAmount, sectorAge abi.ChainEpoch, rewardEstimate, networkQAPowerEstimate *smoothing.FilterEstimate, qaSectorPower abi.StoragePower) abi.TokenAmount {
+// replacedDayReward and replacedSectorAge are the day reward and age of the replaced sector in a capacity upgrade.
+// They must be zero if no upgrade occurred.
+func PledgePenaltyForTermination(dayReward, replacedDayReward, twentyDayRewardAtActivation abi.TokenAmount, sectorAge, replacedSectorAge abi.ChainEpoch, rewardEstimate, networkQAPowerEstimate *smoothing.FilterEstimate, qaSectorPower abi.StoragePower) abi.TokenAmount {
```

Comment:
There are lots of parameters here, and I think we can re-order them for the type checker to help us reduce the chance of errors. Move the replacedDayReward and replacedSectorAge to the end, so their order can't be confused with the subject sector.